include/darknet.h

```diff
@@ -1032,6 +1032,7 @@ LIB_API void diounms_sort(detection *dets, int total, int classes, float thresh,
 // network.h
 LIB_API float *network_predict(network net, float *input);
 LIB_API float *network_predict_ptr(network *net, float *input);
+LIB_API float *network_predict_gl_texture(network *net, uint32_t texture_id);
```

Comment:
Should this be behind an `#ifdef GPU` or similar gate?

Reply:
isn't CUDA_OPENGL_INTEGRATION defined only if GPU is defined too?

Reply:
Yep, I think this comment was left before I added the `CUDA_OPENGL_INTEGRATION` ifdef, so it should be covered now.